docs/reference/filters/group_by/index.md

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
I noticed the file structure is `docs/reference/filters/group_by/index.md`. 

Would it make more sense to have this as a single markdown file that isn't nested in another directory, i.e. `docs/reference/filters/group_by.md`?

Same for the money filter, e.g. `docs/reference/filters/money.md`.